examples/vdb/hana/bookmark/main.tf

```diff
@@ -0,0 +1,140 @@
+terraform {
+  required_providers {
+    delphix = {
+      version = "VERSION"
+      source  = "delphix-integrations/delphix"
+    }
+  }
+}
+
+provider "delphix" {
+  tls_insecure_skip = true
+  key               = "1.XXXX"
+  host              = "HOSTNAME"
+}
+
+resource "delphix_vdb" "example" {
+  bookmark_id             = ""
+  name                    = "vdb_to_be_created"
+  source_data_id          = "dsource-name"
+  vdb_restart             = true
+  environment_id          = "env-name"
+  environment_user_id     = "environment_user_name"
+  target_group_id         = "group-123"
+  snapshot_policy_id      = "test_snapshot_policy"
+  database_name           = "dbname_to_be_created"
+  mount_point             = "/var/mnt"
+  auto_select_repository  = true
+  retention_policy_id     = "test_retention_policy"
+  custom_env_files = ["/export/home/env_file_1"]
+  custom_env_vars = {
+    MY_ENV_VAR1 = "$HOME"
+    MY_ENV_VAR2 = "$CRS_HOME/after"
+  }
+  repository_id = ""
+  appdata_source_params = jsonencode({
+    mountLocation    = "/mnt/bkmrk"
+    configServices = [{
+        "sTenantServicePort" =  "indexserver:30049"
+      },
+      {
+        "sTenantServicePort" =  "xsengine:30052"
+      }]
+    tDatabaseName   = "tfbkmrk"
+    tSystemUserName = "SYSTEM"
+    tSystemUserPassword ="Delphix_123"
+  })
+  config_params jsonencode({
+    processes = 150
+  })
+  appdata_config_params jsonencode({
+    param = "value"
+  })
+  additional_mount_points = [{
+      shared_path = "/",
+      mount_path = "/work",
+      environment_id = "environment-123"
+  }]
+  make_current_account_owner = true
```

Comment:
Move tags and `make_current_account_owner` to the bottom. This applies to every example.

The hooks are related to the VDB itself, so they should be closer to the VDB config.

The tags and `make_current..` are more related to DCT configuration and permissions.

Reply:
done